docs/source/narr/parecer-aberto.rst

```diff
@@ -158,47 +163,50 @@ Exemplo:
-        </sec>
-    </body>
+            <sec>
+                <title>Reviewer</title>
+                <p>Vivamus elementum sapien tellus, a suscipit elit auctor in. Cras est nisl, egestas non ultrices ut, fringilla eu magna. Morbi ullamcorper et diam a elementum. Phasellus vitae diam eget arcu dignissim ultrices. Mauris tempor orci metus, a finibus augue viverra id. Phasellus vitae metus quis metus ultrices venenatis. Integer risus massa, sodales in luctus eget, facilisis at ante. Aliquam pulvinar elit venenatis libero auctor vestibulum.</p>
+                <p>Sed in laoreet sem. Morbi vel imperdiet magna. Curabitur a velit maximus, volutpat metus in, posuere sem. Etiam eget lacus lorem. Nulla facilisi. Phasellus in mi urna. Donec finibus, erat non pharetra dignissim, arcu neque vestibulum enim, vel mollis orci nisl sit amet mauris. Nullam ac iaculis leo. Morbi lobortis arcu velit, at aliquet metus faucibus id.</p>
+            </sec>
+        </body>
         ...
     </sub-article>
+    ...
 
 
 .. _elemento-parecer-exemplo-3:
 
-OPR como link externo
-----------------------
-
-O OPR pode estar publicado em outro site; neste caso, deve-se usar a publicação do parecer como link externo. Esta modalidade também pode ocorrer como :ref:`elemento-article` (Recomendado) ou :ref:`elemento-sub-article`. Para isso considerar uma das regras mencionadas acima, mais:
+Parecer como link externo
+-------------------------
 
+O parecer pode estar publicado em outro site e, neste caso, deve-se referenciá-lo 
+como link externo. Esta modalidade também pode ocorrer como :ref:`elemento-article` 
+(Recomendado) ou :ref:`elemento-sub-article`. Para isso considerar uma das regras 
+mencionadas acima, mais:
 
  * ``@object-type`` com valor ``"referee-report"``;
  * ``@xlink:href`` com a URL do parecer (desde o https://);
  * ``@ext-link-type`` com valor ``"uri"``.
 
+
 Exemplo:
 
 .. code-block:: xml
 
-
     ...
     <body>
         <sec>
             <title>Reviewer</title>
-            <p>This report can be read on:<related-object object-type="referee-report" ext-link-type="uri" xlink:href="https://publons.com/publon/000000/#review-2020xxx">Publons</related-object>
-            </p>
+            <p>This report can be read on:<related-object object-type="referee-report" ext-link-type="uri" xlink:href="https://publons.com/publon/000000/#review-2020xxx">Publons</related-object></p>
         </sec>
     </body>
-     ...
+    ...
 
 
 .. note::
- * Quando o parecer não considerar autoria explícita utilizar ``"<anonymous/>"`` + ``"<role>"`` com atributo ``"@specific-use"`` com os valores: ``"reviewer"`` ou ``"editor"``. 
+ * Quando o parecer não apresentar autoria explícita utilizar ``<anonymous/>`` 
+   + ``<role>`` com atributo ``@specific-use`` com os valores ``reviewer`` ou 
+   ``editor``. 
  * É obrigatório o uso de DOI próprio para publicação de parecer. 
  * Fonte: MENDES-DA-SILVA, (2019); SOUZA, (2017) e OLIVEIRA, (2018).
```

Comment:
Essa Fonte não é mais verdadeira pois o texto mudou. Sugiro retirar.

Reply:
Devo retirar apenas o trecho "Fonte: MENDES-DA-SILVA, (2019); SOUZA, (2017) e OLIVEIRA, (2018).", ou também a frase "É obrigatório o uso de DOI próprio para publicação de parecer."?

Reply:
Creio que a @RPostalli se referia apenas à fonte do texto, que foi modificado no início da documentação.

Reply:
Só a fonte parágrafo: Fonte: MENDES-DA-SILVA, (2019); SOUZA, (2017) e OLIVEIRA, (2018).